src/Component/page/Home/Home.css

```diff
@@ -57,6 +57,7 @@
   grid-template-rows: 2/3;
   height: 3px;
   width: 25%; /* border width */
+  margin-top: 8rem;
```

Comment:
I would normally try to pick one unit of measurement and stick to it wherever possible rather than mixing and matching px, width and rem. Matter of personal taste tho